### PR TITLE
Move VMware to master

### PR DESCRIPTION
**Pull request by VeryEarly**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/VMware/custom/New-AzVMwarePrivateCloud.ps1 [hunk 1]
```diff
@@ -185,12 +190,35 @@ function New-AzVMwarePrivateCloud {
...
+                    'Y' {
+                        Break
+                    }
+    
+                    Default {
+                        Return
+                    }
+                }
+            } catch {
+                throw
```
Comment (dingmeng-xue) [inline src/VMware/custom/New-AzVMwarePrivateCloud.ps1:210]: What error message will be shown up? Could you add an example on your PR description?

File: src/VMware/custom/New-AzVMwarePrivateCloud.ps1 [hunk 2]
```diff
@@ -185,12 +190,35 @@ function New-AzVMwarePrivateCloud {
     )
     
     process {
+
+        if(!$PSBoundParameters.ContainsKey('AcceptEULA')){
+            try {
+                $legalTermPath = Join-Path $PSScriptRoot -ChildPath "LegalTerm.txt"
+                $legalTerm = (Get-Content -Path $legalTermPath) -join "`r`n"
+
+                $confirmation = Read-Host $legalTerm"`n[Y] Yes  [N] No  (default is `"N`")"
```
Comment (dingmeng-xue) [inline src/VMware/custom/New-AzVMwarePrivateCloud.ps1:199]: Could you add an example on your PR description?

Comment (VeryEarly) [reply src/VMware/custom/New-AzVMwarePrivateCloud.ps1:199]: something like unable to read the legal term file

File: src/VMware/custom/New-AzVMwarePrivateCloud.ps1 (hunk 1 above)
Comment (VeryEarly) [reply src/VMware/custom/New-AzVMwarePrivateCloud.ps1:210]: wrap raw exception

Comment (dingmeng-xue) [reply src/VMware/custom/New-AzVMwarePrivateCloud.ps1:210]: https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-exceptions?view=powershell-7.1#re-throwing-a-new-exception

Please follow this example and wrap system exception

File: src/VMware/custom/LegalTerm.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+LEGAL TERMS
+Azure VMware Solution is an Azure Service licensed to you as part of your Azure subscription and subject to the terms and conditions of the agreement under which you obtained your Azure subscription (https://azure.microsoft.com/support/legal/). The following additional terms also apply to your use of AVS:
```
Comment (dingmeng-xue) [inline src/VMware/custom/LegalTerm.txt:2]: Legal term should define the meaning of AVS explicitly. One simple way is "Azure VMware Solution (AVS) is ......"

File: src/VMware/generated/runtime/BuildTime/Cmdlets/ExportPsd1.cs [hunk 4]
```diff
@@ -12,180 +12,114 @@
...
-                File.WriteAllText(Psd1Path, sb.ToString());
-            }
-            catch (Exception ee)
-            {
-                Console.WriteLine($"${ee.GetType().Name}/{ee.StackTrace}");
-                throw ee;
-            }
+            sb.AppendLine($@"{Indent}{Indent}{Indent}Prerelease = {previewVersion}");
         }
+        sb.AppendLine($@"{Indent}{Indent}{Indent}Tags = {"Azure ResourceManager ARM PSModule VMware".Split(' ').ToPsList().NullIfEmpty() ?? "''"}");
```
Comment (erich-wang) [inline src/VMware/generated/runtime/BuildTime/Cmdlets/ExportPsd1.cs:102]: Is `.NullIfEmpty() ?? "''"` necessary?

File: src/VMware/custom/New-AzVMwarePrivateCloud.ps1 [hunk 5]
```diff
@@ -185,12 +190,35 @@ function New-AzVMwarePrivateCloud {
...
+    
+                    Default {
+                        Return
+                    }
+                }
+            } catch {
+                throw
+            }
+        }else {
+            $null = $PSBoundParameters.Remove('AcceptEULA')
```
Comment (erich-wang) [inline src/VMware/custom/New-AzVMwarePrivateCloud.ps1:213]: This line should be in `if` block?

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: src/VMware/generated/runtime/BuildTime/Cmdlets/ExportPsd1.cs (hunk 4 above)
Comment (VeryEarly) [reply src/VMware/generated/runtime/BuildTime/Cmdlets/ExportPsd1.cs:102]: these logic are generated

File: src/VMware/custom/New-AzVMwarePrivateCloud.ps1 (hunk 5 above)
Comment (VeryEarly) [reply src/VMware/custom/New-AzVMwarePrivateCloud.ps1:213]: > This line should be in `if` block?

this is the else for the if above.

**Pull request closed without merging**